packages/experience-editor/src/components/displayRules.tsx

```diff
@@ -355,6 +361,53 @@ export const DisplayRulesSection: React.FC<DisplayRulesSectionProps> = ({
                   </Stack>
                 </Stack>
               )}
+
+              {fieldValueContains(displayConditions.id, "dateRange") && (
+                <Stack spacing={3}>
+                  <SectionHeader
+                    variation="secondary"
+                    headline={"Date Range to display the widget?"}
+                    description={
+                      "Select the start and end date and times for the widget to display."
+                    }
+                  />
+                  <Stack spacing={spacing} direction={"row"}>
+                    <LocalizationProvider dateAdapter={AdapterDateFns}>
+                      <DatePickerInput
+                        field={dateRangeStart}
+                        visible={formFieldVisibility[dateRangeStart.id]}
+                        formValues={formValues}
+                        handleChange={handleChange}
+                      />
+                    </LocalizationProvider>
+                    <Box sx={{ alignSelf: "center" }}>to</Box>
+                    <LocalizationProvider dateAdapter={AdapterDateFns}>
+                      <DatePickerInput
+                        field={dateRangeEnd}
+                        visible={formFieldVisibility[dateRangeEnd.id]}
+                        formValues={formValues}
+                        handleChange={handleChange}
+                      />
+                    </LocalizationProvider>
+                  </Stack>
+                  <FormGroup sx={{ display: "block" }}>
+                    <FormControlLabel
+                      control={
+                        <Checkbox
+                          checked={formValues[dateRangeEnd.id] === null}
+                          onChange={(event) =>
+                            handleChange(
```

Comment:
we should hide the end date when this checkbox is active instead of just deleting the value

Reply:
What if I disable it instead? 🤔